staff/sys/ocf-kubernetes-deploy

```diff
@@ -61,20 +76,30 @@ def main():
     ).check_returncode()
 
     # Bindings passed into the kubernetes-deploy templates
-    bindings = {
-        'version': args.appversion,
-    }
 
-    # kubernetes-deploy requires that this environmental variable be set, and passes
-    # it to templates as the `current_sha` variable. We don't ever use it.
-    os.environ['REVISION'] = 'unused'
+    # Created with 600 perms
+    bindings_file = tempfile.NamedTemporaryFile(suffix='.yaml')
```

Comment:
You should use a `with` statement here, see https://github.com/ocf/utils/blob/352f6eb68434d73cf21acd80dee81dad4543abdc/staff/sys/ssh-list#L51-L56